fs/shm/Kconfig

```diff
@@ -3,16 +3,16 @@
 # see the file kconfig-language.txt in the NuttX tools repository.
 #
 
-config FS_SHM
+config FS_SHMFS
```

Comment:
can we keep FS_SHM as before?

Reply:
SHMFS is consistent with other FS config flags (ROMFS ....) Let's rather keep it consistent

Reply:
Ok.